src/ol_openedx_chat_xblock/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[project]
+name = "ol-openedx-chat-xblock"
+version = "0.1.0"
+description = "An Open edX xBlock to add Open Learning AI chat"
+readme = "README.rst"
+license = {text = "BSD-3-Clause"}
+authors = [
+    {name = "MIT Office of Digital Learning", email = "odl@mit.edu"},
+]
+keywords = ["Python", "edx"]
+requires-python = ">=3.11"
+
+dependencies = [
+    "django>=3.2",
+    "edx-django-utils>=1.3.0",
+    "djangorestframework>=3.14.0",
+    "xblock>=1.9.0",
+    "xblock-utils>=1.2.0",
+]
+
+[project.urls]
+"Homepage" = "https://github.com/mitodl/edx-extensions"
+
+[project.entry-points."xblock.v1"]
+ol_openedx_chat_xblock = "ol_openedx_chat_xblock.block:OLChatXBlock"
+
+[project.entry-points."lms.djangoapp"]
+ol_openedx_chat_xblock = "ol_openedx_chat_xblock.apps:OLOpenedxChatXBlockConfig"
+
+[project.entry-points."cms.djangoapp"]
+ol_openedx_chat_xblock = "ol_openedx_chat_xblock.apps:OLOpenedxChatXBlockConfig"
+
+[tool.hatch.build.targets.wheel]
+packages = ["ol_openedx_chat_xblock"]
+include = [
+  "ol_openedx_chat_xblock/**/*.py",
+  "ol_openedx_chat_xblock/static/**/*",
+  "ol_openedx_chat_xblock/templates/**/*",
+  "ol_openedx_chat_xblock/conf/**/*"
+]
+exclude = [
+  "ol_openedx_chat_xblock/**/tests/"
+]
```

Comment:
Why is it that we are excluding tests from the generated builds here in chat xblock but we are not doing that for all other packages. I think we should keep it consistent.

Reply:
I reviewed all of the pyprojects and removed the excludes where they aren't necessary. I think the only package that still has any test files inside it is the edx_sysdamin, the rest are in neighboring directories and don't need an exclude directive.